dev-app/angular.json

```
@@ -0,0 +1,89 @@
+{
+  "$schema": "../node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": ".",
+  "projects": {
+    "dev-app": {
+      "projectType": "application",
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss",
+          "standalone": true,
+          "changeDetection": "OnPush"
+        }
+      },
+      "root": ".",
+      "sourceRoot": "src",
+      "prefix": "dev-app",
+      "architect": {
+        "build": {
+          "builder": "@angular/build:application",
+          "options": {
+            "externalDependencies": ["path"],
+            "outputMode": "static",
+            "outputPath": "dist",
+            "index": "src/index.html",
+            "browser": "src/main.ts",
+            "server": "src/main.server.ts",
+            "polyfills": [],
+            "tsConfig": "tsconfig.app.json",
+            "inlineStyleLanguage": "scss",
+            "assets": ["src/favicon.ico"],
+            "styles": ["src/app/theme.scss"],
+            "scripts": [],
+            "webWorkerTsConfig": "tsconfig.worker.json",
+            "optimization": {
+              "fonts": {
+                "inline": false
+              }
+            }
+          },
+          "configurations": {
+            "production": {
+              "outputHashing": "all"
+            },
+            "development": {
+              "optimization": false,
+              "extractLicenses": false,
+              "sourceMap": true
+            }
+          },
+          "defaultConfiguration": "production"
+        },
+        "serve": {
+          "builder": "@angular/build:dev-server",
+          "configurations": {
+            "production": {
+              "buildTarget": "dev-app:build:production"
+            },
+            "development": {
+              "buildTarget": "dev-app:build:development"
+            }
+          },
+          "defaultConfiguration": "development",
+          "options": {
+            "buildTarget": "web-ui:build",
+            "headers": {
+              "Cross-Origin-Opener-Policy": "same-origin",
+              "Cross-Origin-Embedder-Policy": "require-corp"
+            }
+          }
+        },
+        "test": {
+          "builder": "@angular/build:unit-test",
+          "options": {
+            "runner": "karma",
+            "browsers": ["ChromeHeadlessNoSandbox"],
+            "include": ["src/app/**/*.spec.ts"]
+          }
+        }
+      }
+    }
+  },
+  "cli": {
+    "analytics": "dca119a9-da31-47f7-a6cb-b60541037021",
```

This UUID needs to be unique and not the same as adev.

```suggestion
    "analytics": "9e881a10-df45-4154-82d7-a3d4a5c351bd",
```